src/runtime/getPositions.ts

```diff
@@ -305,9 +306,20 @@ export async function getPositions(
   // First get all position definitions for the given address
   const definitions = await Promise.all(
     Object.entries(hooksByAppId).map(([appId, plugin]) =>
-      plugin.getPositionDefinitions(network, address).then((definitions) => {
-        return definitions.map((definition) => addAppId(definition, appId))
-      }),
+      plugin.getPositionDefinitions(network, address).then(
+        (definitions) => {
+          return definitions.map((definition) => addAppId(definition, appId))
+        },
+        (err) => {
+          // In case of error, log and return empty array
+          // so other positions can still be resolved
+          logger.warn(
```

Comment:
should we make this an error so that we get alerted about it?

Reply:
Good point. I've updated it to the `error` severity. We won't automatically get alerted by it though, since the API call will succeed.
But we should probably add one specific alert for this.